### PR TITLE
make min resync period configurable

### PR DESCRIPTION
**Pull request by haiker2011**
Signed-off-by: sunhaizhou <sunhaizhou@kingsoft.com>

### CODE REVIEW

File: pkg/globalmanager/config/config.go [hunk 1]
```diff
@@ -58,6 +59,10 @@ type ControllerConfig struct {
 
 	// kb config to info the worker
 	KB KBConfig `json:"knowledgeBaseServer,omitempty"`
+
+	// period config min resync period
+	// default 30s
+	MinResyncPeriod int64 `json:"minResyncPeriod,omitempty"`
```
Comment (llhuii) [inline pkg/globalmanager/config/config.go:65]: Better to specify the second unit, see https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#units

```suggestion
	MinResyncPeriodSeconds int64 `json:"minResyncPeriodSeconds,omitempty"`
```

**Comment by llhuii** [comment]
lgtm, and please squash the commits

**Comment by haiker2011** [comment]
@llhuii  already squash the commits

**Pull request merged**